rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -431,14 +469,51 @@ def _build_tf_train_graph(self) -> Tuple["tf.Tensor", "tf.Tensor"]:
             self.scale_loss,
         )
 
-    def _build_tf_pred_graph(
-        self, session_data: "train_utils.SessionData"
-    ) -> "tf.Tensor":
+    def batch_to_input(self, batch: Tuple) -> Tuple[tf.Tensor, tf.Tensor]:
+        """Convert batch input into correct tensors.
+
+        As we do not know what features (sparse and/or dense) were used, we need to
+        check what features are provided and parse them accordingly.
+        """
+        # batch contains 1 or 2 a_in values, b_in, label_ids
```

Comment:
but response selector could have 1 or 2 b_in values (dense and/or sparse)

Reply:
good point, will have a look

Reply:
are you sure? looking at the code it seems like, we just substitute intent_ids with response_ids, everything else seems to be the same.

Reply:
yes because before there were only one set of features, now we have dense and sparse, so we need to do the same thing, we did for text features in embedding intent classifier